pkg/contracts/test/osmotic-pool/SupportProjects.t.sol

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.17;
+
+import "forge-std/StdMath.sol";
+
+import "@oz/utils/Strings.sol";
+import {MimeToken} from "mime-token/MimeToken.sol";
+
+import {OsmoticPoolSetup} from "../setups/OsmoticPoolSetup.sol";
+
+import {ProjectSupport, SupportUnderflow} from "../../src/OsmoticPool.sol";
+import {ProjectNotInList} from "../../src/interfaces/IProjectList.sol";
+
+contract OsmoticPoolSupportProjects is OsmoticPoolSetup {
+    event ProjectSupportUpdated(uint256 indexed round, uint256 indexed projectId, address participant, int256 delta);
+
+    function testFuzz_SupportProjects(int32[MAX_ACTIVE_PROJECTS] memory _supports) public {
+        ProjectSupport[] memory projectSupports =
+            _processFuzzedSupports(MIME_HOLDER0, _supports, MAX_ACTIVE_PROJECTS, true, false);
```

Comment:
What do you think of define the last two parameters (e.i. onlyPositiveSupports, onlyNonZeroSupports) with variables to provide more clarity?